web/src/styles/global.css

```diff
@@ -66,6 +66,14 @@ img {
   width: 100%;
   height: 100%;
 }
+.image-overlay .image-timestamp {
+  position: absolute;
+  top: 5px;
+  left: 5px;
+  padding: 0 5px;
+  color: var(--gray-dark1);
+  background-color: var(--gray-light3);
```

Comment:
Could you make the background match the close button background (white)?

Reply:
The close button is actually `var(--gray-light2)`

Reply:
Made them match